src/dotnet/Users.Contracts/Backend/IUserProfilesBackend.cs

```diff
@@ -3,7 +3,17 @@ namespace ActualChat.Users;
 public interface IUserProfilesBackend
 {
     [ComputeMethod(KeepAliveTime = 10)]
-    Task<UserProfile?> Get(string userId, CancellationToken cancellationToken);
+    Task<UserProfile?> Get(string userProfileOrUserId, CancellationToken cancellationToken);
```

Comment:
Are you sure it's good to do this instead of two different methods?

Reply:
inFact userProfileId is the same as userId. I can rename it to userId

Reply:
renamed to id